stylesheets/site-structure.scss

```diff
@@ -12,6 +12,10 @@
     margin-right: 20px;
 }
 
+.text-center {
```

Comment:
Huh, no wonder I couldn't find that class in our CSS! I was pretty sure I had used it but couldn't figure out from where.